extensions/variable-width-instruction-standard.md

```diff
@@ -17,9 +17,10 @@ Instruction prefixes are a sequence of one or more bytes which modify the instru
 ### Defined Prefix Sequences
 
 
-| Prefix      | Description        |
-|:------------|:-------------------|
-| `1010 xxxx` | Unused             |
-| `1100 xxxx` | Expanded registers |
-| `1101 xxxx` | Unused             |
+| Prefix      | Description                                                  |
+|:------------|:-------------------------------------------------------------|
+| `1010 xxxx` | When `xxxx` is neither `1110` nor `1111`, conditional prefix |
+| `1010 1110` | Unused                                                       |
```

Comment:
Shouldn't `1010 1111` also be listed here?

Reply:
`1010 1111` isn't a prefix. It's the first byte of the register-based call and jump instructions.